packages/zipkin-instrumentation-express/index.d.ts

```diff
@@ -0,0 +1,22 @@
+import {Tracer} from "zipkin"
+import {Handler} from "express"
+
+/**
+ * When a request comes in, creates ServerRecv annotation and then passes
+ * the request to the next middlewhare. When the final middleware finishes
+ * the request, creates ServerSend annotation
+ *
+ * Sets the tracer.id to the span id prior to running the next middleware.
+ *
+ * Note that if the next middleware makes async calls, it should either
+ * store the span id manually or use a CLSContext so that the annotations
+ * go to the correct spans
+ */
+export declare function expressMiddleware(
+  options: {tracer: Tracer, serviceName?: string, port?: number}
```

Comment:
made a note about possibly removing serviceName from here as the `localServiceName` and `localEndpoint` arg for Tracer ends in less repetition.

Reply:
Should I make a note?

Reply:
maybe take a look at https://github.com/openzipkin/zipkin-js/pull/159#discussion_r151887803? I am just wondering if we should document the serviceName param as it probably should be gone